### ISSUE TITLE
Installation does not clone the latest version of the code

### ISSUE THREAD

**Issue by SeismoFelix** [noisepy/NoisePy#321]
Hello, 

I followed these installation instructions:

```
conda create -n noisepy -y python=3.10 pip
conda activate noisepy
pip install noisepy-seis
```

Then I tried to follow the [noisepy_monitoring_demo notebook](https://seisscoped.org/HPS-book/chapters/noise/noisepy_monitoring_demo.html) . 

However, I got an error in cell corresponding to Step 3: Look at invidual traces and make dv/v measurements. When it comes to The config parameters for time-lapse seismic velocity measurement, it seems that `intb_interval_base` attribute it is not defined in the class `ConfigParameters_monitoring`. I checked directly on the [monitoring_utils.py](https://github.com/noisepy/NoisePy/blob/9fd81c361ec0f86157f99608f04c986e898c7eb9/src/noisepy/monitoring/monitoring_utils.py#L59) in Github and indeed the attribute is defined. But it is not in the code I cloned following the installation instructions. 

The monitoring_utils.py I got with the installation lacks of the following attributes:

>     intb_interval_base: float = Field(
>         default=0.01, description="interval base of intrinsic absorption parameter for a grid-searching process"
>     )
>     mfp_interval_base: float = Field(
>         default=0.2, description="interval base of mean free path for a grid-searching process"
>     )

Since those attributes are used in tutorials I assumed they belong to the latest "stable" version of NoisePy. How can I install the code version that contains the latest modifications of the code that are compatible with the Notebook tutorials?

Thanks for your help

**Comment by kuanfufeng** [comment]
Hi Felix, 

Thank you for pointing out this issue. We released a new version ([v0.9.88](https://github.com/noisepy/NoisePy/releases/tag/v0.9.88)) and updated the [noisepy monitoring notebook](https://seisscoped.org/HPS-book/chapters/noise/noisepy_monitoring_demo.html). You should be able to get the latest version by following the installation instructions.

Please let us know if there are any issues while using it. Thanks.

**Comment by SeismoFelix** [comment]
Terrific Kuan-Fu!

I tested the monitoring notebook and it is working!

Thank you so much for your kind help.
